addons/rating/models/rating_mixin.py

```diff
@@ -40,7 +40,9 @@ class RatingParentMixin(models.AbstractModel):
         compute='_compute_rating_percentage_satisfaction', compute_sudo=True, search='_search_rating_avg')
     rating_avg_percentage = fields.Float('Average Rating (%)', groups='base.group_user',
         compute='_compute_rating_percentage_satisfaction', compute_sudo=True)
-    rating_last_value = fields.Float('Rating Last Value', groups='base.group_user', related='rating_ids.rating')
+    rating_last_value = fields.Float(
+        'Rating Last Value',
+        groups='base.group_user', related='rating_ids.rating', domain="[('consumed', '=', True)]")
```

Comment:
Did not know domain was working on a related on a 2many field ^^ cool :D

Reply:
Boarf, it actually works, but validation of field attributes is screaming on the runbot 😅